docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,31 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+Amazon DynamoDB is available for use in AWS by default. You do not need to set up anything manually to use it.
+
+### Optional steps
+
+* Configure advanced monitoring services with the [Azure official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/monitoring-automated-manual.html), monitoring is enabled in DynamoDB by default.
+* You can change the `Read/Write Capacity` of DynamoDB tables based on your requirements.
+
+Note:-
+
+* Scalar DB/DL schema loader will configure the same Read/Write Capacity for all tables.
+
+## AWS RDS for MySQL, PostgreSQL, Oracle, SQL Server
+
+In this section, you will create an AWS RDS for MySQL/PostgreSQL/Oracle/SQL Server.
```

Comment:
Is this explanation needed? Nothing different from the title and not consistent with the DynamoDB explanation.

Reply:
>Is this explanation needed? Nothing different from the title and not consistent with the DynamoDB explanation.

Yes. I have added a similar statement in the database document for Azure. Regarding DynamoDB, we don't have to configure anything. So I didn't add a similar statement there.
I have also slightly modified the statement.

Reply:
Then, why no such explanation for DynamoDB?